### ISSUE TITLE
Node feat dim of ZINC dataset

### ISSUE THREAD

**Issue by FenyuHu** [graphdeeplearning/benchmarking-gnns#16]
Hello,thank you for this wonderful work.
I print the size of batch_x  in "train_molecules_graph_regression.py",  and its size is [batch_num, 1]. But in Table 10 in the paper, the dimension is 28 for ZINC dataset. Is there anything wrong with this dataset ?  Looking forward to your reply, thanks ！ :)

**Comment by vijaydwivedi75** [comment]
Hi @FenyuHu, thank you for your interest in our work.

The input node features are integers. There are 28 unique integer values being used as node features in ZINC. Each integer denotes an atom. Consider the node features in ZINC dataset as 28-dim one-hot vectors, with each vector representing an atom (the integer).

The code uses [PyTorch's Embedding() layer.](https://pytorch.org/docs/stable/_modules/torch/nn/modules/sparse.html#Embedding) The node integer features (dim 1, but 28 unique values) are input to the `nn.Embedding()` layer (as shown [here](https://github.com/graphdeeplearning/benchmarking-gnns/blob/master/nets/molecules_graph_regression/gat_net.py#L33)) and then to the GNN.

Hope this clarifies.

**Comment by FenyuHu** [comment]
Thanks for your explanation ! It solves my question.
